lib/modules/subredditManager.js

```diff
@@ -176,6 +176,7 @@ module.go = async () => {
 	}
 
 	await getLatestShortcuts();
+	await getMultis();
```

Comment:
so for the `getMultis` function, I was just copying the pattern I saw for `getSubreddits`. I guess I'm unsure if you want a rewrite of the function, move the call to the function to load the multis, or both.